bundler.d/dhcp_isc.rb

```diff
@@ -1,6 +1,10 @@
 group :dhcp_isc_inotify do
   gem 'rsec'
-  gem 'rb-inotify'
+  if RUBY_VERSION < '2.2'
+    gem 'rb-inotify', '< 0.9'
+  else
+    gem 'rb-inotify'
+  end
```

Comment:
Version 0.9.x still works with 2.0 ruby according to rubygems.org so this would do it:

```ruby
  if RUBY_VERSION < '2.1'
    gem 'rb-inotify', '~> 0.9.10'
  else
    gem 'rb-inotify'
  end
```

Reply:
In theory, this allows me to use `rb-inotify` 0.8 on Ruby 2.2+, but not on Ruby 2.1-, updated the pin from `'< 0.9'` to `'< 0.10'`

Reply:
hm, my version does this, too... :man_shrugging: